js/graticule.js

```diff
@@ -299,8 +299,13 @@ export function Graticule (paramMetricQHistoryReference, paramEle, ctx, offsetDi
     // console.log("x-axis labeling offset: " + (this.graticuleDimensions[1] + this.graticuleDimensions[3] + this.pixelsBottom /2));
     for (let i = 0; i < yAxisSteps.length; ++i) {
       if (yPositions[i] >= this.graticuleDimensions[1]) {
-        const textWidth = this.ctx.measureText(yAxisSteps[i][1]).width
-        this.ctx.fillText(yAxisSteps[i][1], this.graticuleDimensions[0] - textWidth - this.pixelsLeft, yPositions[i] + 4)
+        this.ctx.textAlign = 'right'
+        let value = yAxisSteps[i][1]
+        if (value >= 10000) {
+          value = value.toString().substr(0, 1) + '.' + value.toString().substr(1, 1) + ' × 10^' + Math.floor(Math.log(value) / Math.log(10))
```

Comment:
I think we might as well use `log10`[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/log10]